src/Viz.js

```diff
@@ -886,6 +902,16 @@ function value(d) {
     return arguments.length ? (this._messageStyle = assign(this._messageStyle, _), this) : this._messageStyle;
   }
 
+  /**
+      @memberof Viz
+      @desc Sets the inner HTML of the status message that is displayed when no data is supplied to the visualization. Must be a valid HTML string or a function that, when passed this Viz instance, returns a valid HTML string.
+      @param {Function|String} [*value*]
+      @chainable
+  */
+  noDataHTML(_) {
+    return arguments.length ? (this._noDataHTML = typeof _ === "function" ? _ : constant(_), this) : this._noDataHTML;
+  }
+
```

Comment:
still need to add a `noDataMessage` method

Reply:
Okay! I'll add it. I assumed that you wanted it to be kept as an internal variable because that was your suggestion for `dataRequired` and `noDataMessage` is the new name for `dataRequired`.

Reply:
Ah I see. Yeah we still want this external method, as users may want to disable the behavior themselves in favor of a custom solution.